docs/source/linchpincli_linchpin_up.rst

```diff
@@ -0,0 +1,35 @@
+linchpin up
+===========
+
+Usage: ``linchpin up [OPTIONS] TARGET``
+
+Provisions nodes from the given target(s) in the given PinFile.
+
+**Arguments**
+
+``TARGET ...``
+	Provision ONLY the listed target(s). If omitted, ALL targets are provisioned.
```

Comment:
Let's clarify, a user may have multiple PinFiles. Maybe it should read 'If omitted, ALL targets in the appropriate PinFile are provisioned.'